public/app/features/variables/query/QueryVariableRefreshSelect.tsx

```diff
@@ -14,14 +15,15 @@ const REFRESH_OPTIONS = [
 ];
 
 export function QueryVariableRefreshSelect({ onChange, refresh }: PropsWithChildren<Props>) {
+  const { width } = useWindowSize();
```

Comment:
Should we use [`useMediaQueryChange`](https://github.com/grafana/grafana/blob/43f7309272d623e2a3367d29780c68041e81c09a/public/app/core/hooks/useMediaQueryChange.ts#L4) hook? This will only trigger a re-render when the media query changes, not when the window size changes.

Reply:
loved it. didn't know it